src/main/java/net/poweredbyhate/wildtp/WildTPListener.java

```diff
@@ -42,26 +39,29 @@ private void onPreTeleport(PreWildTeleportEvent ev) {
 
     @EventHandler
     private void onPostTele(PostWildTeleportEvent ev) {
-        Location lok   = ev.wildLing.getLocation();
-        World    plaza = lok.getWorld();
+        Location lok = ev.wildLing.getLocation();
+        World plaza = lok.getWorld();
 
         TeleportGoneWild.cure(ev.wildLing);
 
-        if (ev.wc.doCommandz)
-        {
+        if (ev.wc.doCommandz) {
             String p = ev.wildLing.getName(), x = "" + lok.getBlockX(), z = "" + lok.getBlockZ();
 
-            for (String c : ev.wc.commando) Bukkit.dispatchCommand(Bukkit.getConsoleSender(),
-                    c.replace("%PLAYER%", p).replace("%X%", x).replace("%Z%", z));
+            for (String c : ev.wc.commando)
+                Bukkit.dispatchCommand(Bukkit.getConsoleSender(),
+                        c.replace("%PLAYER%", p).replace("%X%", x).replace("%Z%", z));
         }
 
+
         String sound = WildTP.instace.getConfig().getString("Sound");
         if (sound == null || sound.isEmpty())
             return;
 
         try
         {
-            plaza.playSound(lok, Sound.valueOf(sound), 1, 1);
+            String volume = WildTP.instace.getConfig().getString("Volume");
+            plaza.playSound(lok, Sound.valueOf(sound), SoundCategory.MASTER, Float.parseFloat(String.valueOf(volume)), 1);
+            WildTP.debug("Playing da sound of " + sound + " with volume of " + volume);
```

Comment:
ah, I see I already did have a debug in the catch. Heh.

Reply:
> > an option to kill your players ears while teleporting.
> 
> If only. Anything greater than 1 just increases radius not dB unfortunately.

Huh, gotta update that then.